app/views/layouts/listnav/_persistent_volume.html.haml

```diff
@@ -0,0 +1,21 @@
+- if @record.try(:name)
+  #accordion.panel-group
+    = miq_accordion_panel(truncate(@record.name, :length => truncate_length), true, "icon") do
+      = render(:partial  => "layouts/quadicon",
+               :locals   => {:mode => :icon,
+               :item     => @record,
+               :size     => 72,
+               :typ      => :listnav})
+
+    = miq_accordion_panel(_("Properties"), false, "persistent_volume_prop") do
+      %ul.nav.nav-pills.nav-stacked
+        %li
+          = link_to(_('Summary'), {:action => 'show', :id => @record, :display => 'main'}, :title => _("Show Summary"))
+
+    = miq_accordion_panel(_("Relationships"), false, "persistent_volume_rel") do
+      %ul.nav.nav-pills.nav-stacked
+        - if role_allows(:feature => "ems_container_show") && !@record.parent.nil?
+          %li
+            = link_to("#{ui_lookup(:table => "ems_container")}: #{@record.parent.name}",
+                {:controller => "ems_container", :action => 'show', :id => @record.parent.id.to_s},
+                :title => _("Show this container volume's parent %s") % ui_lookup(:table => "ems_container"))
```

Comment:
@zeari, I think you're talking about the `link_to` calls.  Same thing as above ... since all the listnav pages use this, it's probably better to have a single refactoring to handle this differently.  At a first glance via google, it looks like this is not completely trivial to replace.